### PR TITLE
Implemented loading circle

### PR DESCRIPTION
**Pull request by rong4188**

### CODE REVIEW

**Comment by Mooophy** [comment]
Hi @rong4188 
Nice idea for this job. 

Just suspect that the file `Spinner.less` has been forgot to include with this pull request. Seems that this file is not presented in your repo neither. So I guess you probably forgot to use the following command to add this file before committing at local repo:
```ruby
#When new files added into a repo, use this line to add them first.
git add .

#Then perform committing:
git commit -a -m 'added Spinner.less'
```
If I'm not wrong, can you execute the commands above first, before give me a pull request again?

As for the implementation, I like the idea, adding `use strict;` and `===` in place of `==`, etc. As a discussion, however, IMHO the promises chaining perhaps could lead to extra overhead. Because, to my knowledge, `$promise` is supposed to be async method. When chained together, they are going to be executed as sync method. Say the 3 queries take `t1`,`t2`,`t3` to perform respectively. The whole time overhead becomes:
```ruby
T = t1 + t2 + t3
```

As an idea, can we use the following way to get the 'hasLoaded' property?

```js
r0.query(function(){
//..
}).$promise.then(function({
    r0.hasLoaded = true;
}));

r1.query(function(){
//..
).$promise.then(function({
    r1.hasLoaded = true;
}));

r2.query(f).$promise.then(function({
    r2.hasLoaded = true;
}));

// other codes

while( ! [r0, r1, r2].every(function(){ 
 return r.done === true;
 }));
$scope.hasLoaded = true;
```

If not wrong the time overhead can reduce to 
```
T = max(t1,t2,t3)
```

Since my JS experience is very limited, I just list my thought here and discuss with you.

File: MeetU/MeetU/Scripts/app/meetu-meetup.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 //  used for template file : ~/Meetups/Index.cshtml.
 //
 (function () {
-
+    "use strict";
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/app/meetu-meetup.js]: you may want to add you review here in the diff page. It is way easier to trace the issue

File: MeetU/MeetU/Scripts/app/meetu-meetup.js [hunk 2]
```diff
@@ -14,24 +14,31 @@
...
+            Meetup.query(function(data) {
                 $scope.meetupViews = data;
-            });
-            Userview.query(function (userViews) {
-                $scope.userId = userViews[0].userId;
-                $scope.userName = userViews[0].userName;
-            });
-            CommentView.query(function (data) {
-                $scope.allCommentViews = data;
+            }).$promise.then(function() {
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/app/meetu-meetup.js]: I agree your solution about time overhead. I am just wondering if the http request is sent one by one? if it is, that the overhead would not be much different. But anyway, your solution is better, I will do that.

File: MeetU/MeetU/MeetU.csproj [hunk 3]
```diff
@@ -262,6 +262,7 @@
     <Content Include="Content\bootstrap-theme.min.css.map" />
     <Content Include="Content\bootstrap-theme.css.map" />
     <None Include="Content\hello-world.less" />
+    <Content Include="Content\Spinner.less" />
```
Comment (rong4188) [inline MeetU/MeetU/MeetU.csproj]: yes I think I forgot to add the new file. I thought git commit -am"" would do that for me.

File: MeetU/MeetU/Scripts/app/meetu-meetup.js (hunk 1 above)
Comment (Mooophy) [inline MeetU/MeetU/Scripts/app/meetu-meetup.js]: Ye..I'd go with this way next time.

**Comment by Mooophy** [comment]
Closed and wait for the renewed PR that containing the file missed.

**Pull request closed without merging**